modules/shared_options.py

```diff
@@ -113,6 +113,7 @@
     "disable_mmap_load_safetensors": OptionInfo(False, "Disable memmapping for loading .safetensors files.").info("fixes very slow loading speed in some cases"),
     "hide_ldm_prints": OptionInfo(True, "Prevent Stability-AI's ldm/sgm modules from printing noise to console."),
     "dump_stacks_on_signal": OptionInfo(False, "Print stack traces before exiting the program with ctrl+c."),
+    "interrupt_after_current": OptionInfo(False, "Interrupt generation after current image is finished on batch processing"),

```

Comment:
Default to `False` for now, to avoid behavior changes.

But honestly, I personally would have this always on. Would you consider making it default to true?